execution_workflows/APAlyzer/APAlyzer_snakemake/config/config.APAlyzer.yaml

```diff
@@ -0,0 +1,27 @@
+# Configuration file for APAlyzer snakemake pipeline
+
+# Note: paths in this config file are relative to the
+#       working directory APAlyzer_snakemake
+
+# Path to run-specific sample file
+sample_file: "config/samples.csv"
+
+# Path to output directory for run. Default: results/
+out_dir: "results/"
+# Output file name, ends with .tsv
+out_filename: "differential_challenge_output.tsv"
+# Name of log directory to be placed in out_dir. Default: logs/
+logs_subdir: "logs/"
+
+# Path to GTF annotation file
+gtf: "../../../tests/test_data/gencode_2genes_noChr.vM18.annotation.gtf"
+# GTF annotation file organism, genome version, and ensemble version details
+gtf_organism: "Mus_musculus"
+gtf_genome_version: "GRCm39"
+gtf_ensemble_version: "103"
+
+# Read cutoff, default: 0
```

Comment:
A sentence about this parameter "read cutoff" might be helpful. What does it mean?